src/main/resources/application.conf

```diff
@@ -392,12 +392,18 @@ mantis {
     # then: the block will not be queued (such already queued blocks will be removed)
     max-queued-block-number-ahead = 10
 
-    # Maxium number of blocks, after which block hash from NewBlockHashes packet is considered ancient
+    # Maximum number of blocks, after which block hash from NewBlockHashes packet is considered ancient
     # and peer sending it is blacklisted
     max-new-block-hash-age = 20
 
-    # Maxium number of hashes processed form NewBlockHashes packet
+    # Maximum number of hashes processed form NewBlockHashes packet
     max-new-hashes = 64
+
+    # This a recovery mechanism for the issue of missing state nodes during blocks execution:
+    # off - missing state node will result in an exception
+    # on - missing state node will be redownloaded from a peer and block execution will be retried. This can repeat
+    #      several times until block execution succeeds
+    redownload-missing-state-nodes = off
```

Comment:
Please add to sync.conf also.

Reply:
why isn't it "on" by default?

Reply:
@LukasGasior1 because it's a patch not a solution. I'd like for us to be aware the problem still occurs and have motivation to finally fix it.

@mcsherrylabs is it something we want advertise to the users?

Reply:
We need to have this 'on' for release. It can be 'off' in dev. But when this happens the user doesn't want to have to discover a problem (the wallet screen will just pause on 99.98 percent) and then go look online for a fix.

Reply:
So:
- `on` by default, not shown in `sync.conf`
- `off` by default, overridden in `sync.conf`
- or... ?

Reply:
I'd say "on by default, not shown in sync.conf"

Reply:
:+1: